libs/legi-shared/select-option-sort/select-option-sort.pipe.ts

```diff
@@ -8,20 +8,40 @@ import { getLangStringValue, Nil, SelectOption } from '@cognizone/model-utils';
 export class SelectOptionSortPipe implements PipeTransform {
   constructor(private i18nService: I18nService) {}
 
+  /**
+   * `transform` sorts the list of options based on provided sortOrder: `'asc' | 'desc'` and sortType `'label' | 'value'`
+   *
+   * @param {value: Nil<SelectOption[]>, sortType: Nil<'label' | 'value'>, sortOrder: 'asc' | 'desc' = 'asc'}
+   * @returns SelectOption[]
+   */
   transform(value: Nil<SelectOption[]>, sortType: Nil<'label' | 'value'>, sortOrder: 'asc' | 'desc' = 'asc'): SelectOption[] {
     if (!value || !sortType) return value ?? [];
     const mult = sortOrder === 'asc' ? 1 : -1;
     const sorter = sortType === 'value' ? this.valueSort(mult) : this.labelSort(mult);
     return [...value].sort(sorter);
   }
 
+  /**
+   * `labelSort` sorts list of options using localeCompare for values
```

Comment:
```suggestion
   * sorts list of options using localeCompare for values
```
copy/paste 💥 😁 (no need to name whatever we are describing anyway